### ISSUE TITLE
Display EXIF metadata for JXL in Windows Explorer

### ISSUE THREAD

**Issue by VcSaJen** [saschanaz/jxl-winthumb#26]
Windows Explorer displays JPEG metadata using Details Pane in main window or Details tab in Properties window.
It seems possible to add it for custom format using Property Handlers: [https://learn.microsoft.com/en-us/windows/win32/properties/building-property-handlers](https://learn.microsoft.com/en-us/windows/win32/properties/building-property-handlers)

**Comment by saschanaz** [comment]
(Property Handlers implementation is there in https://github.com/saschanaz/jxl-winthumb/blob/main/src/properties.rs, what's not there is reading EXIF there.)

**Comment by supermimai** [comment]
I agree with this request, there is a real need to support IPTC and EXIF in JPEG XL support, I m ready to sponsor this $$

**Comment by supermimai** [comment]
sorry @saschanaz  you redirected me about the question about wic / iptc / exif in windows file explorer, the reply is still not very clear ( not a programmer sorry ) . from my test, the exif aren't displayed  ( my test reference is Adobe Bridge or ExiftoolGui ) 

![image](https://github.com/user-attachments/assets/a16937cd-b4d3-4c0a-a69d-95d552c4b0b9)

when you replied : 
Property Handlers implementation is there in https://github.com/saschanaz/jxl-winthumb/blob/main/src/properties.rs, what's not there is reading EXIF there. 

what ( as a end user ) supposed to do ?

**Comment by saschanaz** [comment]
I'm afraid that as a user there's not much to do. I'm not sure what to do with a dev either, I requested help: https://github.com/tirr-c/jxl-oxide/issues/381
